packager/packager.cc

```diff
@@ -209,9 +209,8 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
     }
 
     bool ownsSymbol(const core::GlobalState &gs, core::SymbolRef symbol) const {
-        auto file = symbol.loc(gs).file();
-        auto &pkg = gs.packageDB().getPackageForFile(gs, file);
-        return this->mangledName() == pkg.mangledName();
+        auto &pkg = gs.packageDB().getPackageNameForSymbol(gs, symbol);
```

Comment:
Two minor nits (pick either or neither to do):
- inline `pkg` and change the `return` to be `return this->mangledName() == gs.packageDB().getPackageNameForSymbol(gs, symbol);`
- rename `pkg` to `pkgName`